var/spack/repos/builtin/packages/py-snakemake-executor-plugin-slurm/package.py

```diff
@@ -16,6 +16,11 @@ class PySnakemakeExecutorPluginSlurm(PythonPackage):
 
     license("MIT")
 
+    version("0.11.2", sha256="fca29038d78387c237afb4ec2b04638e6128d2456940dd7b96ac4205f319d7f3")
+    version("0.11.1", sha256="293a951dcf829400bf6197705d6c602faed95aaaae9ad0d661d23b0184074134")
+    version("0.11.0", sha256="bce1df57900da71175c1c384dbc8f04d8bf8572717c0aaf95c32945a4c7a08d3")
+    version("0.10.2", sha256="919beb114785545f3cc187988f9257a183f1a2c76593e8a8559b87962ebd2651")
+    version("0.10.1", sha256="0d117e3b6fb523b873bd04b7e0117a0c83c9b38dfef1b0a79535c7a084e982ea")
```

Comment:
Version `0.10.1` added a requirement

```python
snakemake = "^8.20.0"
```

Reply:
Same as above. https://github.com/snakemake/snakemake-executor-plugin-slurm/blob/main/pyproject.toml#L28

Reply:
Ok, let's skip it.